src/ConcreteOperations/linear_combination.jl

```diff
@@ -0,0 +1,37 @@
+export linear_combination
+
+"""
+    linear_combination(P1::SimpleSparsePolynomialZonotope, P2::SimpleSparsePolynomialZonotope)
+
+compute the linear combination of simple sparse polynomial zonotopes `P1` and `P2`.
+
+### Input
+
+- `P1` -- simple sparse polynomial zonotope
+- `P2` -- simple sparse polynomial zonotope
+
+### Output
+
+linear combination of `P1` and `P2`.
+
+### Notes
+
+The linear combination of two sets ``P₁`` and ``P₂`` is defined as
+
+```math
+\\{λp₁ + (1-λ)p₂ | p₁ ∈ P₁, p₂ ∈ P₂, λ ∈ [0, 1]\\}
+```
+"""
+function linear_combination(P1::SimpleSparsePolynomialZonotope, P2::SimpleSparsePolynomialZonotope)
```

Comment:
Can you define some auxiliary variables `c1 = center(P1)` and `c2 = center(P2)`, and same for the matrices? I think then the formulas become more readable, eg. we can write `G = 1/2 * [c1-c2  G1  G1  G2  -G2]` as in paper.